setup/jest.config.js

```diff
@@ -52,7 +52,7 @@ module.exports = {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
       '<rootDir>/tests/__mocks__/fileMock.js',
   },
-  collectCoverage: true,
+  collectCoverage: false,
```

Comment:
whats the reason for turn off the collectCoverage?

Reply:
Sorry. Will revert it.